user_guide_src/source/libraries/time/042.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use CodeIgniter\I18n\Time;
+
+$current = Time::parse('2024-03-31', 'Europe/Madrid');
```

Comment:
This is not related to this PR, but I recently read the documentation for **Time::class**.
The point is that dates in the database are commonly stored as `yyyy-MM-dd HH:mm:ss`, but there is no example of using this form in the document. I think you can use `2024-03-31 00:00:00` instead of `2024-03-31` .

Reply:
I sent #8668